src/main/java/org/folio/sidecar/service/routing/EgressRequestHandler.java

```diff
@@ -122,10 +122,10 @@ private boolean requireSystemUserToken(RoutingContext rc) {
   }
 
   private void forwardRequest(RoutingContext rc, HttpServerRequest rq, String moduleId, String url,
-    String updatedPath) {
+                              String updatedPath) {
```

Comment:
This change is not required, looks like that you don't add idea configuration from code uniformity page